src/components/Sidebar.vue

```diff
@@ -2,8 +2,11 @@
   <section class="sidebar">
     <h1 class="banner">Host Console</h1>
     <section class="menu">
-      <router-link to="/" class='menu-row'>
-        <HappsIcon class="home-icon" />hApps
+      <router-link to="/dashboard" class='menu-row' active-class="active-link">
+        <HomeIcon class="home-icon" :color="$route.path.match('/dashboard') ? '#00CAD9' : '#313C59'" />Dashboard
+      </router-link>      
+      <router-link to="/happs" class='menu-row' active-class="active-link">
+        <HappsIcon class="home-icon" :color="$route.path.match('/happs') ? '#00CAD9' : '#313C59'" />hApps
```

Comment:
What's the point of this color-changing logic?

Reply:
It highlights the link you are on.

Reply:
Specifically, the icon. Because the way the icons work, they need the color passed in as a variable, not just taken from a class.

Reply:
Oh I see. `$route.path.match` is checking the current URL. Cool.